### ISSUE TITLE
Parallel bootstrapping

### ISSUE THREAD

**Issue by rofinn** [juliangehring/Bootstrap.jl#36]
It'd be nice if there was a `bootstrap` method (or `pbootstrap` function) that supported distributing the `for`-loop over multiple cores.

**Comment by juliangehring** [comment]
Yes, I agree that would be nice to have. I had that idea of parallelising the bootstrapping on my mind for quite a while, but both going through `pmap` or a parallel loop didn't seem to be a good fit. Maybe it's time to reevaluate this - I'm open for any good ideas.

**Comment by Boxylmer** [comment]
Any luck or news on this? I can definitely add my +1 because some of our fits take minutes and doing 1000 of them is *rough* if it cannot be distributed.
